Extension/src/logger.ts

```diff
@@ -96,6 +97,13 @@ export function getDiagnosticsChannel(): vscode.OutputChannel {
     return diagnosticsChannel;
 }
 
+export function getSshChannel(): vscode.OutputChannel {
+    if (!sshChannel) {
+        sshChannel = vscode.window.createOutputChannel(localize("c.cpp.ssh.channel", "{0}: SSH", "Cpptools"));
```

Comment:
Can this be "C/C++ SSH" to match the naming style of the other logging windows?

Reply:
I would like it to match SSH targets view (Cpptools: SSH Targets), since as we discussed earlier, this feature can be used for other languages.